src/game/models/mod.rs

```diff
@@ -5,10 +5,13 @@ pub mod bullet;
 pub mod asteroid;
 
 use std::ops::{Add, AddAssign, Rem, RemAssign, Sub, SubAssign};
-
+use std::f64;
 use opengl_graphics::GlGraphics;
 use piston_window::{Context, UpdateArgs, Size};
+use std::f64::consts::PI;
```

Comment:
I'm generally trying to follow the "PEP8" style of grouping imports like so, with a single newline in between the groups.

```
use std::
use std::
use std::

use third_party::
use third_party::

use internal::
use internal::
```

I find it makes it a little easier to see at a glance what kind of dependencies a module has.